solutions/cses-1132.mdx

```diff
@@ -80,7 +83,98 @@ int main() {
 	}
 	return 0;
 }
+```				 
+</CPPSection>
+	
+<JavaSection>
+	
+<Warning>
+Java exceeds the time limit on two test cases.
+</Warning>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class TreeDistancesI {
+	// source -> https://cses.fi/problemset/task/1132/
```

Comment:
Do you need this? It's just a link to the problem

Reply:
It is to keep consistent with the java solution.

Reply:
Then remove both occurrences, there's no reason to include it when the link to the problem already appears on the solution page.

Reply:
^Agree with ben

Reply:
agree